spec/api_spec.rb

```diff
@@ -21,10 +21,10 @@ class Bar; end
       end
 
       context "when setting custom options" do
-        before { subject.setup foo: { bar: "baz" }, url: "https://api.example.com" }
+        before { subject.setup foo: { bar: "baz" }, url: "https://api.example.com", timeout: 10, open_timeout: 2 }
```

Comment:
Believe there's confusion around the `timeout` and `open_timeout` options introduced in #400. These options should be part of the [request hash](https://github.com/lostisland/faraday/blob/a712938071bb0997410b5de4a4f2517d30fc4d3d/lib/faraday/options.rb#L192) set on the connection. There's a [thread](https://github.com/lostisland/faraday/issues/417#issuecomment-223413386) on Faraday w/ more detail.

As far as I can tell `open_timeout` and `timeout` are never recognized by Faraday unless they're part of the request hash, going as far back as `0.5.7`. The syntax should be

```ruby
Her::API.setup(url: "https://api.example.com", request: { timeout: 10, open_timeout: 2 })
```

Can you confirm?